### DISCUSSION TITLE
[Telegram] Support send alert to group

### DISCUSSION THREAD

**Discussion by 0123hoang** [jertel/elastalert2#794] [Q&A] [answered]
I can send alert to specific user, but when change `telegram_room_id` to group id, it return error

```
ERROR:elastalert:Error while running alert telegram: Error posting to Telegram: 400 Client Error: Bad Request for url: https://api.telegram.org/bot[...]/sendMessage. Details: {"ok":false,"error_code":400,"description":"Bad Request: chat not found"}
```

**Comment by nsano-rururu** [comment, marked as answer]
How should I change it if it corresponds to the group id? If you know the URL of the reference, please let us know.
https://github.com/jertel/elastalert2/blob/master/elastalert/alerters/telegram.py#L42

**Reply by 0123hoang** [reply]
Oh my bad, sorry.
I'm forgot checking adding group/message permission.
